libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -249,6 +249,52 @@ TEST(CppInterface, MacroCompileOptions) {
                   .GetSuccess());
 }
 
+TEST(CppInterface, DisassemblyOption) {
+  shaderc::Compiler compiler;
+  shaderc::CompileOptions options;
+  options.SetDisassemblyMode();
+  const std::string kMinimalShader = "void main(){}\n";
+  shaderc::SpvModule result = compiler.CompileGlslToSpv(
+      kMinimalShader, shaderc_glsl_vertex_shader, options);
+  EXPECT_TRUE(result.GetSuccess());
+  // This should work with both the glslang native disassembly formt and the
+  // SPIR-V tools assembly format.
```

Comment:
Captialize "SPIR-V Tools" since that's a proper project name, and the name of a GitHub repo.